### ISSUE TITLE
Unexpected content type : application/x-gzip (expected: [application/octet-stream])

### ISSUE THREAD

**Issue by durasj** [raydac/mvn-jlink#9]
Hi @raydac,

Thanks for the plugin! It looks like Github is using Content-Type `application/x-gzip` for authorized requests instead of the expected `application/octet-stream` - the body seems to be identical. We're using LIBERICA provider and this causes our builds to fail unnecessarily. Any suggestions or preferences how this could be handled?

Thanks

<details>
  <summary>Relevant part of the stack trace</summary>
  
```
Caused by: java.io.IOException: Unexpected content type : application/x-gzip (expected: [application/octet-stream])
    at com.igormaznitsa.mvnjlink.utils.HttpUtils.doGetRequest (HttpUtils.java:170)
    at com.igormaznitsa.mvnjlink.jdkproviders.AbstractJdkProvider.doHttpGetIntoFile (AbstractJdkProvider.java:270)
    at com.igormaznitsa.mvnjlink.jdkproviders.providers.LibericaOpenJdkProvider.downloadAndUnpack (LibericaOpenJdkProvider.java:170)
    at com.igormaznitsa.mvnjlink.jdkproviders.providers.LibericaOpenJdkProvider.lambda$getPathToJdk$2 (LibericaOpenJdkProvider.java:141)
    at com.igormaznitsa.mvnjlink.jdkproviders.AbstractJdkProvider.loadJdkIntoCacheIfNotExist (AbstractJdkProvider.java:357)
    at com.igormaznitsa.mvnjlink.jdkproviders.providers.LibericaOpenJdkProvider.getPathToJdk (LibericaOpenJdkProvider.java:140)
    at com.igormaznitsa.mvnjlink.mojos.AbstractJdkToolMojo.getSourceJdkFolderFromProvider (AbstractJdkToolMojo.java:251)
    at com.igormaznitsa.mvnjlink.mojos.MvnCacheJdkMojo.onExecute (MvnCacheJdkMojo.java:73)
    at com.igormaznitsa.mvnjlink.mojos.AbstractJdkToolMojo.execute (AbstractJdkToolMojo.java:210)
```
</details>

**Comment by raydac** [comment]
good point, I will fix

**Comment by raydac** [comment]
could you try snapshot?

**Comment by durasj** [comment]
@raydac thank you! Works like a charm! I was previously using this plugin without the authorization, but Github's own IPs (via Github Actions) were always hitting the rate limit. It worked for months before Github started playing with the mime type - first it was `application/zip` and then `application/x-gzip`. Nice to see this sorted out. Do you have any plans on the release? No hurry, just asking.

![image](https://user-images.githubusercontent.com/1062898/142951527-4702f254-bb30-489c-8948-e5b00e9659ed.png)

Thanks again, much appreciated!

**Comment by raydac** [comment]
I have released 1.1.1 version and placed it into maven central so that soon should be accessible
